simapp/sim_test.go

```diff
@@ -31,8 +31,14 @@ import (
 )
 
 func init() {
+	// TODO: remove "Simulation" prefix as it's verbose
```

Comment:
Is there an issue for this? Can we tackle it in this PR if it's not too much overhead?